src/shared/api/OneSignalApiShared.ts

```diff
@@ -21,11 +21,11 @@ export default class OneSignalApiShared {
     return OneSignalApiBase.put(`players/${playerId}`, { app_id: appId, ...options });
   }
 
-  static sendNotification(appId: string, playerIds: Array<string>, titles, contents, url, icon, data, buttons) {
+  static sendNotification(appId: string, onesignalId: string, titles, contents, url, icon, data, buttons) {
     var params = {
       app_id: appId,
       contents: contents,
-      include_player_ids: playerIds,
+      include_aliases: { onesignalId: [onesignalId] },
```

Comment:
This own be allowed without auth.

Reply:
I'm not sure what this means

Reply:
Sorry meant to say `include_aliases` requires auth to use. Specifically only the app's REST API key will work for this. No plans to support a client side JWT for this field.